java/src/test/java/ai/rapids/cudf/TestUtils.java

```diff
@@ -211,4 +214,16 @@ private static boolean hasInf(int v) {
     });
     return result;
   }
+
+  public static File getResourceAsFile(String resourceName) {
+    URL url = TableTest.class.getClassLoader().getResource(resourceName);
+    if (url == null) {
+      throw new IllegalArgumentException("Unable to locate resource: " + resourceName);
```

Comment:
nit: I think `FileNotFoundException` is better in this context

Reply:
The problem with `FileNotFoundException` is that it's a checked exception which makes this utility more annoying to use in practice.  For example, the existing use-cases would need try..catch blocks around them, which in turn would lead to something like a `getResourceAsFileUnchecked` method. :smile: